src/HttpOverStream.Client/DialMessageHandler.cs

```diff
@@ -1,188 +1,188 @@
-                        {
-                            response.Content.Headers.TryAddWithoutValidation(name, value);
-                        }
-                    }
-                    catch (FormatException ex)
-                    {
-                        throw new HttpRequestException("Error parsing header", ex);
-                    }
-                }
-                Debug.WriteLine("Client: Finished reading response header lines");
-                responseContent.SetContent(new BodyStream(stream, response.Content.Headers.ContentLength, closeOnReachEnd: true), response.Content.Headers.ContentLength);
-                return response;
-            }
-            catch(Exception e)
-            {
-                Debug.WriteLine("Client: EXCEPTION:" + e);
-                stream?.Dispose();
-                throw;
-            }
-        }
-
-        private void ParseStatusLine(HttpResponseMessage response, string line)
-        {
-            const int MinStatusLineLength = 12; // "HTTP/1.x 123" 
-            if (line.Length < MinStatusLineLength || line[8] != ' ')
-            {
-                throw new HttpRequestException("Invalid response, expecting HTTP/1.0");
-            }
-
-            if (!line.StartsWith("HTTP/1.0"))
-            {                
-                throw new HttpRequestException("Invalid response, expecting HTTP/1.0");
-            }
-            response.Version = HttpVersion.Version10;
-            // Set the status code
-            if (int.TryParse(line.Substring(9,3), out int statusCode))
-            {
-                response.StatusCode = (HttpStatusCode)statusCode;
-            }
-            else
-            {
-                throw new HttpRequestException("Invalid response, can't parse status code");
-            }
-            // Parse (optional) reason phrase
-            if (line.Length == MinStatusLineLength)
-            {
-                response.ReasonPhrase = string.Empty;
-            }
-            else if (line[MinStatusLineLength] == ' ')
-            {
-                response.ReasonPhrase = line.Substring(MinStatusLineLength + 1);
-            }
-            else
-            {
-                throw new HttpRequestException("Invalid response");
-            }
-        }
-
-        private void ValidateAndNormalizeRequest(HttpRequestMessage request)
-        {
-            request.Version = HttpVersion.Version10;
-            // Add headers to define content transfer, if not present
-            if (request.Headers.TransferEncodingChunked.GetValueOrDefault())
-            {
-                throw new HttpRequestException("DialMessageHandler does not support chunked encoding");
-            }
-
-            // HTTP 1.0 does not support Expect: 100-continue; just disable it.
-            if (request.Headers.ExpectContinue == true)
-            {
-                request.Headers.ExpectContinue = false;
-            }
-        }
-    }
-}
+﻿using System;
```

Comment:
Any idea why this diffs the whole file? It seems to have only a few diffs, right?

Reply:
erf, might be line endings...

Reply:
![image](https://user-images.githubusercontent.com/2386884/60110884-2cc75080-976d-11e9-9836-b1672d1917de.png)

did the trick.

Reply:
Yeah I know, but it's still smelly that it even happens... I'm looking into my git settings.

Reply:
(all text files should be converted to LF on commit normally)

Reply:
I have found the issue and will fix on the next PR. Thanks!